src/query/storages/parquet/src/parquet_rs/parquet_reader/row_group.rs

```diff
@@ -317,3 +348,59 @@ impl<'a> RowGroups for InMemoryRowGroup<'a> {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::sync::Arc;
+
+    use bytes::Bytes;
+    use common_base::base::tokio;
+    use opendal::services::Memory;
+    use opendal::Operator;
+    use parquet::basic::ConvertedType;
+    use parquet::basic::Repetition;
+    use parquet::basic::Type as PhysicalType;
+    use parquet::file::metadata::RowGroupMetaData;
+    use parquet::schema::types::*;
+
+    use crate::parquet_rs::parquet_reader::row_group::InMemoryRowGroup;
+
+    #[tokio::test]
+    async fn test_merge() {
```

Comment:
The test seems only test the read parquet is ok or not, not test the `*_merge_io_read` functions with the ReadSettings?